setup.py

```diff
@@ -50,8 +50,8 @@
     "neo4j": ["py2neo"],
     "test": [
         "pytest==5.3.1",
-        "pytest-benchmark>=3.1",
         "pytest-cov>=2.6.0",
+        "pytest-repeat>=0.8.0",
```

Comment:
I think you deleted the wrong thing 😄

Reply:
hm yeah i think we might still need benchmark 😂